packages/oruga/src/components/breadcrumb/BreadcrumbItem.vue

```diff
@@ -0,0 +1,144 @@
+<script setup lang="ts">
+import { defineClasses /*useProviderChild */ } from "@/composables";
+
+import { getOption } from "@/utils/config";
+
+import { computed, type PropType } from "vue";
+import OIcon from "../icon/Icon.vue";
+
+import type { ComponentClass } from "@/types";
+/**
+ * The classic breadrcumb item, in different colors, sizes, and states
+ * @displayName Breadcrumb Item
+ */
+defineOptions({
+    isOruga: true,
+    name: "OBreadcrumbItem",
+    configField: "breadcrumb",
+    inheritAttrs: true,
+});
+
+const props = defineProps({
+    active: {
+        type: String,
+        default: () => getOption("breadcrumb.active"),
+    },
+    tag: {
+        type: String,
+        default: () => getOption("breadcrumb.tag"),
```

Comment:
Please define the tag as similar to other components:

 tag: {
        type: [String, Object, Function] as PropType<DynamicComponent>,
        default: () => getOption<DynamicComponent>("breadcrumb.tag", "li"),
    },